endpoints/openrtb2/video_auction.go

```diff
@@ -128,11 +128,13 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		cacheTTL = int64(deps.cfg.CacheURL.DefaultTTLs.Video)
 	}
 	debugLog := exchange.DebugLog{
-		Enabled:   strings.EqualFold(debugQuery, "true"),
-		CacheType: prebid_cache_client.TypeXML,
-		TTL:       cacheTTL,
-		Regexp:    deps.debugLogRegexp,
+		Enabled:       strings.EqualFold(debugQuery, "true"),
+		CacheType:     prebid_cache_client.TypeXML,
+		TTL:           cacheTTL,
+		Regexp:        deps.debugLogRegexp,
+		DebugOverride: exchange.IsDebugOverrideEnabled(r.Header.Get(exchange.DebugOverrideHeader), deps.cfg.Debug.OverrideToken),
```

Comment:
To ensure I'm following correctly, we need to perform this check in the video endpoint because the video endpoint does it's own debug logging. The auction and amp endpoints don't require this since the logic in auction.go applies to all 3 openrtb2 endpoints, right?

Reply:
Correct. Now debug log is enabled for video endpoint only. For amp and auction we don't init debugLog and by default it's disabled. func (e *exchange) HoldAuction:
```
if debugLog == nil {
	debugLog = &DebugLog{Enabled: false}
}
```